browser/src/control/AutoCompletePopup.ts

```diff
@@ -136,16 +118,45 @@ abstract class AutoCompletePopup {
 		});
 	}
 
+	getCursorPosition(): Point {
+		const commentSection = app.sectionContainer.getSectionWithName(
+			L.CSections.CommentList.name,
+		);
+
+		if (commentSection?.sectionProperties?.selectedComment?.isEdit()) {
```

Comment:
(bizarrely) the "selectedComment" appears to actually be the *root* comment of the current comment. So if we insert a comment, and then reply to the comment and "ok" and then edit the reply and use an at-mention. The selectedComment is the root comment, which is not actually being edit and the at-mention popup appears in the main document